src/EnergyPlus/Autosizing/CoolingAirFlowSizing.cc

```diff
@@ -80,9 +80,10 @@ Real64 CoolingAirFlowSizer::size(EnergyPlusData &state, Real64 _originalValue, b
             } else if (this->zoneEqSizing(this->curZoneEqNum).DesignSizeFromParent) {
                 this->autoSizedValue = this->zoneEqSizing(this->curZoneEqNum).AirVolFlow;
             } else {
-                auto const SELECT_CASE_var(this->zoneEqSizing(this->curZoneEqNum).SizingMethod(DataHVACGlobals::CoolingAirflowSizing));
-                if ((SELECT_CASE_var == DataSizing::SupplyAirFlowRate) || (SELECT_CASE_var == DataSizing::None) ||
-                    (SELECT_CASE_var == DataSizing::FlowPerFloorArea)) {
+                switch (this->zoneEqSizing(this->curZoneEqNum).SizingMethod(DataHVACGlobals::CoolingAirflowSizing)) {
```

Comment:
I'm not going to harp on this because I want this stuff merged, but this is a good candidate for 
`auto & thisZoneEqSizing = this->zoneEqSizing(this->curZoneEqNum);` etc.

Reply:
Actually, given what we are seeing in the debug builds and the desire to move to `EPVector` I will be harping on this more. :grin: